.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build and Unit Test
+    runs-on: ubuntu-latest
```

Comment:
I'm not as familiar with Github Actions as other CI, but we could probably use a less-heavy runner, something more akin to the `golang` containers on Dockerhub.

Reply:
> I'm not as familiar with Github Actions as other CI, but we could probably use a less-heavy runner, something more akin to the golang containers on Dockerhub.

I'm also super-duper new to Github Actions but it seems like there aren't any light-weight options for the `runs-on` parameter based on [the docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on). FWIW in practice I'm seeing these actions run must faster than the Travis equivalents :shrug:

Reply:
@cpu If there’s a Dockerhub image that does the needful, it’s also real easy to use a Docker step, which will mount the workspace into the image.

My terribly-hacky still WIP [cabforum integration](https://github.com/sleevi/cabf-scratch/blob/main/.github/workflows/build-docs.yml) does this for the Pandoc step, and hopefully next week will have a custom step that builds all the tools as a Docker image to begin with.

Reply:
I'm largely following the [setup-go](https://github.com/actions/setup-go) instructions. I don't believe there's a more tailored docker image to use for this but would be open to pointers if folks know of one.